English_transcripts/school/Epic Quest/sharedAssets/P_C_M_ENG_L3_AU_numline_83.txt

```diff
@@ -0,0 +1 @@
+Look at the number line 8 is to the right of 3. Which number is greater 8 or 3? Touch the correct number.
```

Comment:
Look at the number line**,** 8 is to the right of 3.

Reply:
Updated the transcript.